api_tests/registrations/views/test_registration_list.py

```diff
@@ -1264,6 +1264,52 @@ def test_new_API_version_uses_embargo_end_date(
         assert res.status_code == 201
         assert res.json['data']['attributes']['pending_embargo_approval'] is True
 
+    def test_new_API_version_errors_on_lift_embargo(
+            self, app, user, draft_registration, url_registrations_ver):
+        today = timezone.now()
+        three_years = (
+            today +
+            dateutil.relativedelta.relativedelta(
+                years=3)).strftime('%Y-%m-%dT%H:%M:%S')
+
+        payload = {
+            'data': {
+                'type': 'registrations',
+                'attributes': {
+                    'draft_registration_id': draft_registration._id,
+                    'lift_embargo': three_years
+                }
+            }
+        }
+        res = app.post_json_api(
+            url_registrations_ver,
+            payload,
+            auth=user.auth,
+            expect_errors=True)
+        assert res.status_code == 400
+        assert (res.json['errors'][0]['detail'] == f'The field lift_embargo was '
```

Comment:
parens here are unnecessary, but I'm not sure how they effect line break, so maybe they are, maybe split this over more lines.